internal/driver/credentials.go

```diff
@@ -0,0 +1,35 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package driver
+
+import (
+	"github.com/edgexfoundry/go-mod-bootstrap/bootstrap/secret"
```

Comment:
@brianointel this is where the bad import is coming from. needs to be updated to v3